.github/workflows/update-cudo-catalog.yml

```
@@ -0,0 +1,49 @@
+name: "update-cudo-catalog"
+on:
+  schedule:
+    - cron:  '00 */7 * * *'
```

Increase the minutes a bit to make sure all the actions do not conflict with each other.
```suggestion
    - cron:  '05 */7 * * *'
```